server/planning/planning.py

```diff
@@ -252,6 +252,6 @@ class PlanningResource(superdesk.Resource):
     resource_methods = ['GET', 'POST']
     item_methods = ['GET', 'PATCH', 'PUT', 'DELETE']
     public_methods = ['GET']
-    privileges = {'POST': 'planning',
-                  'PATCH': 'planning',
+    privileges = {'POST': 'planning_planning_management',
+                  'PATCH': 'planning_planning_management',
                   'DELETE': 'planning'}
```

Comment:
Spike should be `PATCH` method not `DELETE`. You are changing the state not deleting an item. 
Also, there is no need for `DELETE` in `item_methods`.